### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by dantman** [react-native-community/react-native-svg#528]
There are a bunch of critical bugfixes that have been merged into master and haven't been published yet (#505 included).

It's probably a good time to release a 6.0.1 bugfix release to npm.

@dustinsavery Are there any issues you'd like to see fixed before the next release.

**Comment by dustinsavery** [comment]
Totally agree, just waiting on a response from @magicismight re: release process before continuing (I've never done a release for this repo yet). Once I have that information, I'll be more on top of rolling out releases.

I'll probably do a beta release first, just to make sure we're stable, before rolling it out to the masses. Sound good?

**Comment by dustinsavery** [comment]
@dantman New pre-release is out! https://github.com/react-native-community/react-native-svg/releases/tag/6.0.1-rc.0

I'll keep this open for a week and if there are no reported issues, we'll push it fully live.

**Comment by dantman** [comment]
Seems to be working so far upgrading to it.

**Comment by nover** [comment]
It still seems to me like it pulls in the `react 0.11` Pod file, which wreaks havoc in my `react-native 0.50` project. Is this intended behavior?

**Comment by dustinsavery** [comment]
@nover no, that is not intended. Can you give a bit more information? I have not seen this issue.

**Comment by nover** [comment]
@dustinsavery 
On the project where we are currently having issues with version 5.5 I tried to clean up, remove and unlink and then install version `6.0.1-rc.1` however it was still linking with `react 0.11` - in a fresh repo it does however seem to work properly, so I will have to dig deeper :)

**Comment by dustinsavery** [comment]
@nover keep us posted, in case others have similar issues.

**Comment by msand** [comment]
New versions have been published, and I haven't seen these issues, nor more reports. Will go ahead and close this for now.
